.github/workflows/ci-test-tlm.yml

```diff
@@ -78,6 +78,7 @@ jobs:
   launch-tlm-test-notif:
     name: Comment TLM test start on PR
     runs-on: ubuntu-latest
+    if: github.event.comment.body == '/test-tlm'  # Only run if the comment is "/test-tlm"
```

Comment:
qq: will this only check for latest comment or any comment?

Reply:
This whole file is ran when a comment is made. `github.event` will check specifically the comment that triggered the event (latest)